splice_machine/src/main/java/com/splicemachine/derby/stream/control/ControlDataSetProcessor.java

```diff
@@ -41,20 +41,38 @@
 import com.splicemachine.storage.Partition;
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 import org.apache.commons.collections.iterators.SingletonIterator;
+//import org.apache.kafka.clients.consumer.CommitFailedException;
+//import org.apache.kafka.clients.consumer.ConsumerConfig;
+//import org.apache.kafka.clients.consumer.ConsumerRecords;
+//import org.apache.kafka.clients.consumer.ConsumerRecord;
```

Comment:
remove them?

Reply:
Fixed in new commit [e15bf8e](https://github.com/splicemachine/spliceengine/pull/3391/commits/e15bf8e4ed58fce49d5a3048e6bca875bad887bd)